README.md

```diff
@@ -46,35 +46,36 @@ gradle release
 ```
 gradle test jacocoTestReport
 ```
-
-
-##Legacy Build (DEPRECATED!)
 ### Regenerate Javadocs
 
 ```
-rm -r <devcenter.home>/content/reference/android/api/*
-cd <project.home> 
-mvn -Pdev javadoc:javadoc install
+<devcenter.home> should be located in this relative path ../../../<devcenter.home> relatively <java-library.home>.
+cd <devcenter.home> 
+git pull devcenter
+cd <java-library.home>
+gradle release
```

Comment:
It would be good to mention that javadocs will be generated in this directory after these steps